lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Transactions.hs

```diff
@@ -90,17 +92,15 @@ spec = do
     it "TRANS_EXTERNAL_CREATE_01 - proper single output transaction and \
        \proper binary format" $ \ctx -> do
         let toSend = 1 :: Natural
-        (ExternalTxFixture _ wDest _ txWits@((Tx txid _ _), _)) <-
+        (ExternalTxFixture _ wDest _ tx) <-
             fixtureExternalTx @t ctx toSend
         let baseOk = Base16
-        let arg = T.unpack $ encodeTx txWits MsgTypeTransaction baseOk
+        let arg = T.unpack $ txToBase tx baseOk
 
         -- post external transaction
-        (Exit code, Stdout out, Stderr err) <-
+        (Exit code, Stdout _out, Stderr err) <-
             postExternalTransactionViaCLI @t ctx [arg]
-        let expectedTxId = T.decodeUtf8 . convertToBase Base16 . getHash $ txid
         err `shouldBe` "Ok.\n"
-        out `shouldBe` "{\n    \"id\": " ++ show expectedTxId ++ "\n}\n"
```

Comment:
Weakened test, think it should be fine though

Reply:
Why though? (generally speaking, the commit already tells the "what", in reviews and in comments, we are interested about the "why", especially when it's not obvious for the reader).

Reply:
Good that you were sceptical. I misunderstood the point of this check when I skimmed through it, and we can indeed keep it. 1fa5d8743fffa121d5361b0c232c2622c6eeed32